packages/button/index.ts

```diff
@@ -69,4 +87,4 @@ function Button (props = {}) {
   `
 }
 
-module.exports = Button
+export = Button
```

Comment:
This is how we'll have to export the converted components until everything has switched to ESM. Once that happens, we can use normal ESM exports. However, this also means that we are limited to a single export per file until then. Just something to keep in mind.